plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -288,23 +290,24 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.CHISELED_RED_SANDSTONE, 0xB7601B);
         put(Blocks.CHISELED_SANDSTONE, 0xD8CB9B);
         put(Blocks.CHISELED_STONE_BRICKS, 0x787778);
+        put(Blocks.CHORUS_FLOWER_DEAD, 0x613E5E);
         put(Blocks.CHORUS_FLOWER, 0xA184A1);
         put(Blocks.CHORUS_PLANT, 0x5D395D);
         put(Blocks.CLAY, 0xA1A6B3);
         put(Blocks.COAL_BLOCK, 0x101010);
         put(Blocks.COAL_ORE, 0x686867);
         put(Blocks.COARSE_DIRT, 0x77553B);
-        put(Blocks.COBBLED_DEEPSLATE, 0x4D4D50);
         put(Blocks.COBBLED_DEEPSLATE_SLAB, 0x4D4D50);
         put(Blocks.COBBLED_DEEPSLATE_STAIRS, 0x4D4D50);
         put(Blocks.COBBLED_DEEPSLATE_WALL, 0x4D4D50);
-        put(Blocks.COBBLESTONE, 0x807F7F);
+        put(Blocks.COBBLED_DEEPSLATE, 0x4D4D50);
         put(Blocks.COBBLESTONE_SLAB, 0x807F7F);
         put(Blocks.COBBLESTONE_STAIRS, 0x807F7F);
         put(Blocks.COBBLESTONE_WALL, 0x807F7F);
+        put(Blocks.COBBLESTONE, 0x807F7F);
         put(Blocks.COBWEB, 0xE6EAEB);
         put(Blocks.COMMAND_BLOCK, 0xAD846E);
-        put(Blocks.COMPARATOR, 0xA39695);
+        put(Blocks.COMPARATOR, 0x000000);
```

Comment:
Redstone items should show up on the map, too. We're showing the red from the wire color, and these pieces take up more block space than wires, do. Would be nice for a redstone technical player to see their contraptions on the map